production_configs/20220303_tuned_ze40_south/lstmcpipe_config.yml

```diff
@@ -0,0 +1,44 @@
+## MC PRODUCTION FULL PIPELINE CONFIGURATION ##
+#
+# Template example to request a MC prod.
+
+
+workflow_kind: lstchain  # Not expected to be changed by the user
+prod_id: zen40az180_dl1ab_tuned_nsb_psf         # ex: local_no_n_islands. Default; v00 (if key left empty or None)
+
+base_path: /fefs/aswg/data/mc  # Path for centralised LST MC prods
+
+# Only for dl1 reprocessing - you will also need to change:
+# `r0_to_dl1` for `dl1ab` in "stages_to_be_run"
+#dl1_reference_id: # ex: 20220215_v0.9.1_prod5_trans_80_local_tailcut_8_4
```

Comment:
You will need to refer to a previous MC prod if you are running the dl1ab stage.

The `dl1_reference_id` should be `20220304_v0.9.3_prod5_trans_80_zen40az180_local_tailcut_8_4`, same reason as here https://github.com/cta-observatory/lstmcpipe/pull/176/files#r819590898

Otherwise all the rest is perfect. We will run this prod soon @jurysek